File: languages/SoftwareComponentLanguage.sandbox/source_gen/SoftwareComponentLanguage/sandbox/Webstore_ComponentTest.cpp
#include <gtest/gtest.h>
#include <memory>
#include "Order/IShoppingInterface.h"
#include "Order/Webstore.h"

class Webstore_ComponentTest : public testing::Test
{
protected:
};

TEST_F(Webstore_ComponentTest, testCase)
{
  std::shared_ptr<Order::Webstore> sut = std::make_shared<Order::Webstore>();
  sut->addProduct(1);
}

File: languages/SoftwareComponentLanguage.sandbox/source_gen/SoftwareComponentLanguage/sandbox/Webstore_ComponentTest.cs
/// <filename>
///     Webstore_ComponentTest.cs
/// </filename>
using Order;
using Microsoft.VisualStudio.TestTools.UnitTesting;

[Microsoft.VisualStudio.TestTools.UnitTesting.TestClass]
public class Webstore_ComponentTest
{
    [Microsoft.VisualStudio.TestTools.UnitTesting.TestMethod]
    public void testCase()
    {
        Order.Webstore sut = new Order.Webstore();
        sut.addProduct(1);
        Assert.IsTrue(true, "Expected that method runs without exception");
    }
}
